Wikimate.php

```diff
@@ -495,23 +567,87 @@ public function getError()
 /**
  * Models a wiki article page that can have its text altered and retrieved.
  *
- * @author  Robert McLeod
+ * @author  Robert McLeod & Frans P. de Vries
  * @since   December 2010
  */
 class WikiPage
 {
+	/**
+	 * Section list identifier for index
+	 *
+	 * @var integer
+	 */
 	const SECTIONLIST_BY_INDEX = 1;
+
+	/**
+	 * Section list identifier for name
+	 *
+	 * @var integer
+	 */
 	const SECTIONLIST_BY_NAME = 2;
+
+	/**
+	 * Section list identifier for number
+	 *
+	 * @var integer
+	 */
 	const SECTIONLIST_BY_NUMBER = 3;
 
-	protected $title          = null;
-	protected $wikimate       = null;
-	protected $exists         = false;
-	protected $invalid        = false;
-	protected $error          = null;
+	/**
+	 * The title of the page
+	 *
+	 * @var string|null
+	 */
+	protected $title = null;
+
+	/**
+	 * Wikimate object for API requests
+	 *
+	 * @var Wikimate|null
+	 */
+	protected $wikimate = null;
+
+	/**
+	 * Does the page exist?
```

Comment:
Same as [above](https://github.com/hamstar/Wikimate/pull/109/files#r665263159):

```suggestion
	 * Whether the page exists
```

(Or something more verbose, like "Indicator of whether the page exists")

Reply:
I'll stick with the concise here :)